client/me/purchases/purchases-list-in-dataviews/purchases-list-header.jsx

```diff
@@ -0,0 +1,29 @@
+import { CompactCard } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+import 'calypso/me/purchases/style.scss';
+
+const PurchasesListHeader = ( { showSite = false } ) => {
```

Comment:
Any chance we could make this file TypeScript? It wouldn't require a lot of extra work since it only has one type. I think it would just be this:

```suggestion
const PurchasesListHeader = ( { showSite = false }: { showSite?: boolean } ) => {
```

Reply:
Done.

Reply:
Follow-up question: Would we type the return HTML as a `CompactCard`?

Reply:
Actually, turns out it won't ever get rendered in the DataView. So I'll just remove the file 😁